srcjs/components/actionSheet.js

```diff
@@ -57,10 +62,8 @@ $(function() {
   Shiny.addCustomMessageHandler("update-action-sheet", function(message) {
     // Destroy old instance
     app.store.state.actions[message.id].destroy();
-    // Create new config
-    var a = app.actions.create(message);
-    // Update app data
-    app.store.state.actions[message.id] = a;
+    // Create new instance including callbacks and click events
+    createActionSheet(message);
```

Comment:
@DivadNojnarg after updating the action sheet, clicking the buttons again didn't do anything. Since we destroy the complete instance first, we also need to make sure that we have the handlers to set inputs on click etc.